### ISSUE TITLE
ability to make a cartocontext.map() without specifying datalayers

### ISSUE THREAD

**Issue by andy-esch** [CartoDB/cartoframes#62]
Sometimes a data scientist just needs to show a map!

**Comment by andy-esch** [comment]
It works, but it needs a zoom and center:
```python
cc.map(layers=[BaseMap()], zoom=5, lat=38, lng=-122)
```

**Comment by dmed256** [comment]
Weird, it should be adding a `BaseMap` if not found

```
        if layers is None:
            layers = []

       ....

        # Check basemaps, add one if none exist
        base_layers = [idx for idx, layer in enumerate(layers) if layer.is_basemap]
        if len(base_layers) > 0:
            ...
        else:
            layers.insert(0, BaseMap())
```

**Comment by dmed256** [comment]
Fixed in e08ab79
